packages/changed-elements-react/public/locales/en/VersionCompare.json

```
@@ -107,8 +107,8 @@
     "noDescription": "No description",
     "scrollToChangedProperty": "Cycle to next changed property",
     "checkingAvailability": "Checking availability",
-    "namedVersionErrorPlaceholder": "<Error>",
-    "noNamedVersions": "No named versions found in iModel to do comparison",
+    "namedVersionErrorPlaceholder": "No named versions found.",
+    "noNamedVersions": "No named versions found in iModel to do comparison.",
```

Very minor suggestion, but this would be a little more concise.
```suggestion
    "noNamedVersions": "No named versions found in iModel for comparison.",
```